source/renderer/app/components/layout/VerticalFlexContainer.scss

```diff
@@ -2,6 +2,5 @@
   display: flex;
   flex-direction: column;
   height: 100%;
-  overflow-y: overlay;
```

Comment:
@DeeJayElly are you sure this change is required and that it doesn't break anything?